src/web/routes/teams/integration.test.ts

```diff
@@ -34,11 +34,104 @@ import {
 const figmaTeamSummaryComparer = (a: FigmaTeamSummary, b: FigmaTeamSummary) =>
 	a.teamId.localeCompare(b.teamId);
 
-const TEAMS_CONFIGURE_ENDPOINT = '/teams/configure';
-const TEAMS_LIST_ENDPOINT = '/teams/list';
+const TEAMS_ENDPOINT = '/teams';
 
 describe('/teams', () => {
-	describe('POST /configure', () => {
+	describe('GET', () => {
```

Comment:
Just moved this block up to match the route definition order in `teams-router.ts`